vsintegration/tests/Salsa/VsMocks.fs

```diff
@@ -1649,10 +1649,10 @@ module internal VsActual =
         let vsvar =
             let var = Environment.GetEnvironmentVariable("VS160COMNTOOLS")
             if String.IsNullOrEmpty var then
-                Environment.GetEnvironmentVariable("VSAPPIDDIR")
+                Environment.GetEnvironmentVariable("VSAPPDDIR")
```

Comment:
errrrrrr wut? So was this ever working?

Reply:
Probably not, but it also likely never actually got here either.